code/game/objects/items/bees_items.dm

```diff
@@ -47,7 +47,7 @@
 /obj/item/weapon/book/manual/hydroponics_beekeeping
 	name = "The Ins and Outs of Apiculture - A Precise Art"
 	icon_state ="bookHydroponicsBees"
-	author = "Beekeeper Nanako"
```

Comment:
why is nanako die for the most generic name ever? just curious

Reply:
Nanako is the name of a previous coder, and the same one who made this bee change.

Reply:
Ah, alright, thanks. Could we get anything but Dave, though? I've seen so many Daves and Bills.

Reply:
Reason Dave was used is that that was the old name.

Reply:
I think that's an example of it being used all the time, but fair enough there.